architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-before-{context}']
+= Before you begin deploying the AWS Service Broker
+
+Before you can get started deploying the AWS Service Broker, make sure you meet the prerequisites described in this topic.
+
+.Prerequisites 
+
+The following items are required to deploy the AWS Service Broker into {product-title}.
+
+{product-title}::
+You must have an {product-title} cluster configured to run the Service Catalog.
+
+////
+AWS Service Broker:: 
+You must have an link:https://aws.amazon.com/partners/servicebroker/[AWS Service Broker] configured with an appropriate registry, such as link:https://hub.docker.com/u/awsservicebroker/[docker.io/awsservicebroker].
+////
+
+AWS Service Instance Policy::
+You must create specific credential and permissions policies in AWS for the AWS Service Broker user or role.
+
+You can create the two required policies using the xref:aws-getting-started-before-policies[provided JSON samples], then assign the policies to the AWS Service Broker user or role or to a separate role, see [Managing Resources Via Assumed Role](/docs/README.md#managing-resources-via-assumed-role).
```

Comment:
Double check this link : /docs/README.md#managing-resources-via-assumed-role

Reply:
I might make this the first step of the main process instead of a prereq.

Reply:
@kalexand-rh Thanks for the reminder. I need to check all the links.